tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc

```diff
@@ -732,6 +732,18 @@ Status ConvertAfterShapes(const ConversionParams& params) {
     segment_options.exclude_node_list.insert(node);
   }
   segment_options.minimum_segment_size = params.minimum_segment_size;
+
+  // We allow the segmenter to include or exclude a node based on the precision
+  // of the TRT engine we will build. We only know the precision of the TRT
+  // engine when calibration is not used. When calibration is used, we may build
+  // a FP32 engine or a INT8 engine for the same segment.
```

Comment:
When I think about this more,  I think we are not sure whether FP32 engine is created. Let's revise the last sentence to this:
When calibration is used, we build an INT8 engine and another engine with precision FP16 or FP32 for the same segment to support calibration.

Reply:
Changing this :)